docs/apache-airflow/howto/docker-compose/index.rst

```diff
@@ -125,7 +125,7 @@ On **all operating systems**, you need to run database migrations and create the
 
 .. code-block:: bash
 
-    docker compose up airflow-init
+    docker-compose up airflow-init
```

Comment:
Is it a mistake?
https://stackoverflow.com/questions/66514436/difference-between-docker-compose-and-docker-compose

Reply:
That's right, i didn't know there is new way to call a docker-compose.yml file. In the rest of the page its been used the 'docker-compose' command instead of 'docker compose' and i thought was a typo.